packages/chain/consensus/batch_proposal.go

```diff
@@ -153,24 +162,33 @@ func (c *Consensus) calcBatchParameters(props []*BatchProposal) (*consensusBatch
 // calcIntersection a simple algorithm to calculate acceptable intersection. It simply takes all requests
 // seen by 1/3+1 node. The assumptions is there can be at max 1/3 of bizantine nodes, so if something is reported
 // by more that 1/3 of nodes it means it is correct
+const keyLen = ledgerstate.OutputIDLength + 32
+
```

Comment:
The godoc comment above applies to the `calcIntersection` function

```suggestion
const keyLen = ledgerstate.OutputIDLength + 32

// calcIntersection a simple algorithm to calculate acceptable intersection. It simply takes all requests
// seen by 1/3+1 node. The assumptions is there can be at max 1/3 of bizantine nodes, so if something is reported
// by more that 1/3 of nodes it means it is correct
```